### PR TITLE
Apiview

### PR DESCRIPTION
**Pull request by 0486**
Play around.....

### CODE REVIEW

File: .settings/org.eclipse.core.resources.prefs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```
Comment (elpiankova) [inline .settings/org.eclipse.core.resources.prefs]: Delete '.settings/org.eclipse.core.resources.prefs' from committed change

File: requirements.txt [hunk 2]
```diff
@@ -5,6 +5,8 @@ django-extensions     # for graph models
 markdown
 django-filter
 jsonfield
+djsw_wrapper==1.8.0
```
Comment (elpiankova) [inline requirements.txt]: Should be `djsw_wrapper>=0.1.9`

Comment (0486) [inline requirements.txt]: Should be 0.1.8 until @lyssdod will not fix the module.

File: .settings/org.eclipse.core.resources.prefs (hunk 1 above)
Comment (0486) [inline .settings/org.eclipse.core.resources.prefs]: +

**Pull request merged**